pkgs/games/humblebundle/default.nix

```
@@ -28,6 +28,7 @@ let
     owlboy = callPackage ./owlboy.nix {};
     pico-8 = callPackage ./pico-8.nix {};
     rocketbirds = callPackage ./rocketbirds.nix {};
+    spacechem = callPackage ./spacechem.nix {};
```

As mentioned in [NixOS/nixpkgs#66620](https://github.com/NixOS/nixpkgs/pull/66620#pullrequestreview-294718277), it seems that the game is i686-only, so why not use `callPackage_i686`?

```suggestion
    spacechem = callPackage_i686 ./spacechem.nix {};
```